[docs] Add info about contribution to docs (#6816)
DevExpress/testcafe: CONTRIBUTING.md

        -   [Create a Minimal Working Example](#create-a-minimal-working-example)
    -   [Ask for Community Support](#ask-for-community-support)
-   [Contribute Code](#contribute-code)
-   [Contribute to Documentation](#contribute-to-documentation)

## The TestCafe Community

> 

Reviewer: Since I'm unable to edit line 31 below, can you please update it for me?

Line 31:
In the last sentence of this paragraph, I don't think "the presence of" is necessary. Can we update this line as follows?

`TestCafe is a [Node.js](https://nodejs.org/en/) application. It supports [all actively maintained](https://github.com/nodejs/Release#release-schedule) versions of the Node.js framework. The installation process requires the [node package manager](https://www.npmjs.com/) (npm) software utility.`

-   Poor quality code.
-   Other [development priorities](https://testcafe.io/402949/roadmap) may take precedence.

Please also note that we do not merge pull requests until the changes are documented. You're welcome to document a new capability yourself. You should add your description as a pull request comment, and include links to Testcafe [documentation](https://testcafe.io/documentation) pages that need editing. While writing the documentation, please, follow the [guidelines](#guidelines).

## Contribute to Documentation

> Testcafe [documentation](https://testcafe.io/documentation) is not open-source. If you find an inaccuracy in the docs, send us a note in a [Github issue](https://github.com/DevExpress/testcafe/issues/new). Please, don't forget to add `AREA: docs` label so that the appropriate team gets a new issue for review.

Reviewer 2: * The name of our product should always be written in UpperCamelCase, unless we're explicitly talking about the `testcafe` npm package.
* The first sentence states a fact that seemingly contradicts the heading, but there's no follow-up. Why is it important that our docs aren't open-source?
* Github issues aren't a communication platform. You can't send notes in issues. Both you and the issue can, however, *notify* us of something. 
```suggestion
Users can't submit documentation pull requests because TestCafe stores [documentation](https://testcafe.io/documentation) in a private repository. To notify the team of an inaccuracy in the docs, create a [Github issue](https://github.com/DevExpress/testcafe/issues/new) with the `AREA: docs` label.
```

#### Guidelines

> We follow a set of documentation guidelines and practices. The main idea is to make our docs easy to read. So, when submitting a new issue, please, make sure that:

Reviewer 2: Who are the "we" in this sentence? I'd rather specify this point, or make 'documentation' the subject of the sentence. "The main idea is" might be too colloquial and definitely doesn't add anything to the paragraph
```suggestion
The technical writing team at Developer Express adheres to a set of guidelines and practices that make our docs easy to read. When you submit a new issue, please, make sure that:
```

-   You don't use gerunds, passive voice and past tense
-   Every sentence expresses a single idea

> -   You don't needlessly repeat the same piece of information

Reviewer 2: I know this doesn't have to be an extensive list, but are we sure that's the right way to end the article? Seems a bit abrupt...

-   Poor quality code.
-   Other [development priorities](https://testcafe.io/402949/roadmap) may take precedence.

> We merge pull requests after the changes are documented. If you want to document a new capability, write a comment with your description to a pull request. Please, follow the [guidelines](#guidelines) during the writing. 

Reviewer 2: * you cannot "write to" a pull request
* a pull request -> *the* pull request because we're talking about specific new capabilities
* the third sentence includes a gerund and an unnecessary introductory word.
```suggestion
We merge pull requests after the changes are documented. If you want to document a new capability, add a comment with the description to the pull request. Review our [writing guidelines](#guidelines) before you proceed.
```

## Contribute to Documentation

Users can't submit documentation pull requests because TestCafe stores [documentation](https://testcafe.io/documentation) in the private repository. To notify the team of an inaccuracy in the docs, create a [Github issue](https://github.com/DevExpress/testcafe/issues/new) with the `AREA: docs` label.

#### Guidelines

> TestCafe team adheres to a set of writing guidelines and practices that make our docs easy to read. When you submit a comment about a new capability or a new issue of an inaccuracy, please, follow the main rules:

Reviewer 2: let's simplify both sentences

```suggestion
TestCafe team adheres to a set of writing guidelines that make our docs easy to read. Follow these rules when you submit written content:
```

> -   You don't use gerunds, passive voice and past tense
> -   Every sentence expresses a single idea
> -   You don't needlessly repeat the same piece of information

Reviewer 2: - these aren't rules
- the list isn't uniform
- this isn't a great selection, we probably need a couple more (maybe it's a subject for a later discussion)

```suggestion
-   Avoid gerunds, passive voice and past tense.
-   Keep your sentences simple. Make sure that each sentence only expresses a single idea.
-   Do not needlessly repeat the same piece of information.
```

-   Poor quality code.
-   Other [development priorities](https://testcafe.io/402949/roadmap) may take precedence.

We merge pull requests after the changes are documented. If you want to document a new capability, add a comment with the description to the pull request. Review our [writing guidelines](#guidelines) before you proceed. 

## Contribute to Documentation

> Users can't submit documentation pull requests because TestCafe stores [documentation](https://testcafe.io/documentation) in a private repository. To notify the team of an inaccuracy in the docs, create a [Github issue](https://github.com/DevExpress/testcafe/issues/new) with the `AREA: docs` label.

Reviewer: ```suggestion
Users cannot submit documentation pull requests because TestCafe stores [documentation](https://testcafe.io/documentation) in a private repository. To notify the team of an inaccuracy in the documentation, create a [Github issue](https://github.com/DevExpress/testcafe/issues/new) with the `AREA: docs` label.
```

#### Guidelines

> TestCafe team adheres to a set of writing guidelines that make our docs easy to read. Follow these rules when you submit written content:

Reviewer: ```suggestion
The TestCafe team adheres to a set of writing guidelines that make our documentation easy to read. Follow these rules when you submit written content:
```

-   Avoid gerunds, passive voice, and past tense.
-   Avoid double negatives.

> -   Avoid ambiguity, weasel words, and jargon.

Reviewer: I think we should update "weasel words" because not all readers may be familiar with this term.

```suggestion
-   Avoid misleading or ambiguous words and jargon.
```

-   Avoid vague and confusing references between pronouns and their antecedents.
-   Be descriptive rather than prescriptive.

> -   Keep your sentences short and simple. Make sure that each sentence only expresses a single idea.

Reviewer: ```suggestion
-   Keep your sentences short and simple. Make sure that each sentence expresses only a single idea.
```

> -   Do not needlessly repeat the same piece of information.

Reviewer: Can we omit the word "needlessly"?

`Do not repeat the same piece of information.`

Alternatively, can we say the following instead?

`Avoid repetition.`

> -   If possible, include interactive examples (codes samples, screenshots, etc).

Reviewer: ```suggestion
-   If possible, include interactive examples (codes samples, screenshots, etc.).
```

> When the suggestions satisfy the above rules, it helps TestCafe team to update the documentation quicker.

Reviewer: ```suggestion
When the suggestions satisfy the above rules, it helps the TestCafe team to update the documentation.
```

> When the suggestions satisfy the above rules, it helps the TestCafe team to update the documentation.

Reviewer 2: let's delete this line — it doesn't add anything

-   Poor quality code.
-   Other [development priorities](https://testcafe.io/402949/roadmap) may take precedence.

We merge pull requests after the changes are documented. If you want to document a new capability, add a comment with the description to the pull request. Review our [writing guidelines](#guidelines) before you proceed. 

## Contribute to Documentation

Users can't submit documentation pull requests because TestCafe stores [documentation](https://testcafe.io/documentation) in a private repository. To notify the team of an inaccuracy in the docs, create a [Github issue](https://github.com/DevExpress/testcafe/issues/new) with the `AREA: docs` label.

#### Guidelines

TestCafe team adheres to a set of writing guidelines that make our docs easy to read. Follow these rules when you submit written content:

-   Avoid gerunds, passive voice, and past tense.
-   Avoid double negatives.
-   Avoid ambiguity, weasel words, and jargon.

> -   Avoid vague and confusing references between a pronoun and its antecedent.

Reviewer 2: ```suggestion
-   Avoid vague and confusing references between pronouns and their antecedents.
```
